secretmanager/snippets.rb

```diff
@@ -260,6 +260,76 @@ def get_secret_version project_id:, secret_id:, version_id:
   version
 end
 
+def iam_grant_access project_id:, secret_id:, member:
+  # [START secretmanager_iam_grant_access]
+  # project_id = "YOUR-GOOGLE-CLOUD-PROJECT"  # (e.g. "my-project")
+  # secret_id  = "YOUR-SECRET-ID"             # (e.g. "my-secret")
+  # member     = "USER-OR-ACCOUNT"            # (e.g. "user:foo@example.com")
+
+  # Require the Secret Manager client library.
+  require "google/cloud/secret_manager"
+
+  # Create a Secret Manager client.
+  client = Google::Cloud::SecretManager.secret_manager_service
+
+  # Build the resource name of the secret.
+  name = "projects/#{project_id}/secrets/#{secret_id}"
+
+  # Get the current IAM policy.
+  policy = client.get_iam_policy resource: name
+
+  # Add new member to current bindings
+  policy.bindings ||= []
```

Comment:
Are you finding that this is necessary? I would hope (and expect) that it wouldn't be.

(But if so, why is it not also necessary in `iam_revoke_access`?)